draft-ietf-tls-tls13.md

```diff
@@ -2560,24 +2583,41 @@ certificate_request_context
   unless used for the post-handshake authentication exchanges
   described in {{post-handshake-authentication}}.
 
-supported_signature_algorithms
-: A list of the signature algorithms that the server is
-  able to verify, listed in descending order of preference. Any
-  certificates provided by the client MUST be signed using a
-  signature algorithm found in supported_signature_algorithms.
+extensions
+: An optional set of extensions describing the parameters of the
+  certificate being requested. The "signature_algorithms"
+  extension MUST be specified.
+{:br}
+
+In prior versions of TLS, the CertificateRequest message
+carried a list of signature algorithms and certificate authorities
+which the server would accept. In TLS 1.3 the former is expressed
+by sending the "signature_algorithms" extension. The latter is
+expressed by sending the "certificate_authorities" extension
+(see {{certificate-authorities}}).
+
+Servers which are authenticating with a PSK MUST NOT send the
+CertificateRequest message.
 
-certificate_authorities
-: A list of the distinguished names {{X501}} of acceptable
-  certificate_authorities, represented in DER-encoded {{X690}} format.  These
-  distinguished names may specify a desired distinguished name for a
-  root CA or for a subordinate CA; thus, this message can be used to
-  describe known roots as well as a desired authorization space.  If
-  the certificate_authorities list is empty, then the client MAY
-  send any certificate that meets the rest of the selection criteria
-  in the CertificateRequest, unless there is some external arrangement
-  to the contrary.
 
-certificate_extensions
+#### OID Filters
```

Comment:
I'm having trouble following the structure, so it might be implied, but you should describe where this can appear.  (Not the ClientHello, I hope).